testing/wlroots/APKBUILD

```diff
@@ -0,0 +1,39 @@
+# Contributor: Henrik Riomar <henrik.riomar@gmail.com>
+# Maintainer:
+pkgname=wlroots
+pkgver=0.2
+pkgrel=0
+pkgdesc="A modular Wayland compositor library"
+url="https://github.com/swaywm/wlroots"
+arch="all"
+license="MIT"
+options="!check" # contains no test suite
+makedepends="ctags
+	eudev-dev
+	libcap-dev
+	libinput-dev
+	libpng-dev
```

Comment:
This shouldn't be required. You can add `-Dexamples=false` to Meson flags to make sure examples aren't built.

Reply:
it still checks for libpng with this diff
```diff
--- a/testing/wlroots/APKBUILD
+++ b/testing/wlroots/APKBUILD
@@ -27,7 +27,7 @@ builddir="$srcdir/$pkgname-$pkgver"

 build() {
        cd "$builddir"
-       meson build --prefix /usr
+       meson build --prefix /usr -Dexamples=false
        ninja -C build
 }
```

```log
Dependency threads found: YES
Dependency wayland-cursor found: YES 1.16.0
Dependency libpng found: YES 1.6.35
Dependency libavutil found: NO (tried pkgconfig)
Dependency libavcodec found: NO (tried pkgconfig)
Dependency libavformat found: NO (tried pkgconfig)
```

Reply:
It checks for dependencies, but doesn't use them. Same for `libav*` for instance.